src/Propel/Generator/Model/ForeignKey.php

```diff
@@ -1019,4 +1019,30 @@ public function isAtLeastOneLocalPrimaryKey()
 
         return 0 !== count($cols);
     }
+
+    public function isTheSamePolymorphicSignature(ForeignKey $fk)
```

Comment:
There should be a discussion around the names introduced in this model as they're going to become highly visible.